modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLManagementToolbarPropsTransformer.js

```diff
@@ -334,6 +334,18 @@ export default function propsTransformer({
 					url: selectFileEntryTypeURL,
 				});
 			}
+			else if (item?.data?.action === 'openExtensionSelector') {
+				openSelectionModal({
+					buttonAddLabel: Liferay.Language.get('select'),
+					height: '70vh',
+					multiple: true,
+					onSelect() {},
+					selectEventName: `${portletNamespace}FilterByExtensions`,
```

Comment:
This `eventName` is probably wrong, maybe should be `selectedFileExtension` as defined [here](https://github.com/liferay-lima/liferay-portal/pull/4068/files#diff-09feee0948001c867b0197bc18a5c99dfb9c93996d5706fa74394574b5639b7fR665). Anyway we can test this in next tasks while handling the onSelect() fn.

Reply:
I have to make some more changes so I include it on this pull!